### ISSUE TITLE
Portable NVDA does not read when brousing to open a file in Audacity

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3179]
**Reported by andrewd on 2013-04-21 23:59**
In Audacity 2.03 on Windows7, NVDA portable version (installed works fine) will not read the list of files.

**Comment by ehollig** [comment]
I am not able to replicate using portable NVDA and Audasity 2.1.2. CC @derekriemer. Any ideas?

**Comment by bhavyashah** [comment]
If the file list in Audacity's Open File dialog is indeed running as administrator as @jcsteh suspects, there probably is nothing that NVDA can do to resolve this issue. Regardless, testing with a temporary copy of a recent NVDA Next snapshot and Audacity 2.1.2, the folder view list was read by NVDA as expected. Since this ticket is more than four years old, the original author of this ticket cannot be contacted, my testing results imply a worksforme, and Jamie's suspicion suggests a cantfix/NeedsExternalFix, I suggest closing. @ehollig

**Comment by leonardder** [comment]
Closing as works for me.
